pkg/collector/corechecks/cluster/topologycollectors/service_collector.go

```diff
@@ -184,12 +184,19 @@ func (sc *ServiceCollector) serviceToStackStateComponent(service v1.Service, end
 		}
 	}
 
+	// add identifier for this service name
+	identifiers = append(identifiers, fmt.Sprintf("urn:service:/%s:%s", sc.GetInstance().URL, serviceID))
+
 	log.Tracef("Created identifiers for %s: %v", service.Name, identifiers)
 
 	serviceExternalID := sc.buildServiceExternalID(serviceID)
 
 	tags := sc.initTags(service.ObjectMeta)
 
+	if service.Spec.ClusterIP == "None" {
+		tags["service"] = "headless"
```

Comment:
this would mean that a service of type `NodePort` would be `headless` but that is not true, right ?